yaml-cpp error

Dear all,
I am getting this error on the last step, when building with `cmake --build .` 
`C:\Users\hach_lu\.conan\data\yaml-cpp\0.6.3\_\_\package\ff82a1e70ba8430648a79986385b20a3648f8c19\include\yaml-cpp/excep
tions.h(148): error C2220: Warnung wird als Fehler interpretiert, es wurde keine object-Datei generiert. [D:\OSP\libcos
im-master-unchanged\libcosim-master\build\src\cosim.vcxproj]
C:\Users\hach_lu\.conan\data\yaml-cpp\0.6.3\_\_\package\ff82a1e70ba8430648a79986385b20a3648f8c19\include\yaml-cpp/excep
tions.h(148): warning C4275: class "std::runtime_error" ist keine DLL-Schnittstelle und wurde als Basisklasse für die D
LL-Schnittstelle class "YAML::Exception" verwendet [D:\OSP\libcosim-master-unchanged\libcosim-master\build\src\cosim.vc
xproj]
`
I am using conan 1.49.0, cmake 3.18.0 and VS 2017. What I am doing wrong here?

--- a/cmake/FindYAML_CPP.cmake
+++ b/cmake/FindYAML_CPP.cmake
@@ -10,7 +10,7 @@
 find_path(YAML_CPP_INCLUDE_DIR NAMES yaml-cpp/yaml.h)
 mark_as_advanced(YAML_CPP_INCLUDE_DIR)
 
-find_library(YAML_CPP_LIBRARY NAMES yaml-cpp libyaml-cpp yaml-cppmd libyaml-cppmd libyaml-cppmdd)
+find_library(YAML_CPP_LIBRARY NAMES yaml-cpp libyaml-cpp  yaml-cppd yaml-cppmd libyaml-cppmd libyaml-cppmdd)
 mark_as_advanced(YAML_CPP_LIBRARY)
 
 include(FindPackageHandleStandardArgs)

--- a/conanfile.py
+++ b/conanfile.py
@@ -20,7 +20,7 @@ class LibcosimConan(ConanFile):
         "fmilibrary/2.3",
         "ms-gsl/2.1.0",
         "libzip/1.7.3",
-        "yaml-cpp/0.6.3",
+        "yaml-cpp/0.7.0",
         "xerces-c/3.2.2",
         # conflict resolution
         "openssl/1.1.1k",

--- a/src/cosim/scenario_parser.cpp
+++ b/src/cosim/scenario_parser.cpp
@@ -17,7 +17,7 @@
 
 #ifdef _MSC_VER
 #    pragma warning(push)
-#    pragma warning(disable : 4127)
+#    pragma warning(disable : 4127 4275)
 #endif
 #include <yaml-cpp/yaml.h>
 #ifdef _MSC_VER

